test/cmocka/src/math/arithmetic/exponential_hifi.c

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2022 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <math.h>
+#include <cmocka.h>
+#include <string.h>
+
+#include <sof/math/exp_fcn_hifi.h>
+#include <sof/audio/format.h>
+#include <rtos/string.h>
+#include <sof/common.h>
+
+#define ULP_TOLERANCE 5.60032793
+#define ULP_SCALE 0.0000999999999749
+/* testvector in Q4.28, -5 to +5 */
+#if (EXPONENTIAL_HIFI3 == 1 || EXPONENTIAL_HIFI4 == 1 EXPONENTIAL_HIFI5 == 1)
+	static const int32_t iv[128] = {
```

Comment:
#if (EXPONENTIAL_HIFI3 == 1 || EXPONENTIAL_HIFI4 == 1 || EXPONENTIAL_HIFI5 == 1)?

Reply:
Yes, will do here as well. Thank you